content/lua/concepts/tables/terms/foreach/foreach.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'foreach() Function'
+Description: 'The foreach() function is a programming construct that iterates through the elements of a provided table or collection, using a specified callback function to perform operations on each element'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Javascript'
+CatalogContent:
+  - 'introduction-to-javascript'
+- 'paths/front-end-engineer-career-path'
+---
+
+## foreach() Function
+
+The `foreach()` function is a programming construct that iterates through the elements of a provided table or collection, using a specified callback function to perform operations on each element. During each iteration, the callback function is invoked and passed two parameters: the index (or key) and the value associated with the current element in the table.
+
+### Syntax  
+```
+foreach(table, callback)
+```
+### Example
+
+Let's consider a practical example of using the `foreach()` function in Lua, a popular programming language. In this example, we have a table containing the names of fruits, and we want to print each fruit name along with its index in the table.
```

Comment:
```suggestion
In this example is a table containing the names of fruits, and it will use `.foreach()` to print each fruit name along with its index in the table.
```
We avoid first and second-person pronouns in docs.